UI/BluetoothExplorer/BluetoothExplorer/BluetoothExplorer/Platforms/Android/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+  <application android:allowBackup="true" android:supportsRtl="true"></application>
```

Comment:
This is missing bluetooth permissions that used to exist in the old AndroidManifest.xml